pkg/syncer/spec/spec_controller.go

```diff
@@ -96,7 +99,47 @@ func NewSpecSyncer(gvrs []schema.GroupVersionResource, upstreamClusterName logic
 				c.AddToQueue(gvr, obj)
 			},
 		})
-		klog.InfoS("Set up informer", "clusterName", upstreamClusterName, "pcluster", workloadClusterName, "gvr", gvr.String())
+		klog.V(2).InfoS("Set up upstream informer", "clusterName", upstreamClusterName, "pcluster", workloadClusterName, "gvr", gvr.String())
+
+		downstreamInformers.ForResource(gvr).Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+			DeleteFunc: func(obj interface{}) {
+				unstrob, ok := obj.(*unstructured.Unstructured)
+				if !ok {
+					runtime.HandleError(fmt.Errorf("unexpected object type: %T", obj))
+					return
+				}
+				klog.V(2).InfoS("downstream delete event", "gvr", gvr, "namespace", unstrob.GetNamespace(), "name", unstrob.GetName())
```

Comment:
Our convention is more like V(3), and add a V(2) with the key above the enqueue. Compare https://github.com/kcp-dev/kcp/blob/main/CONTRIBUTING.md.